config/config.go

```diff
@@ -106,7 +106,15 @@ type Complement struct {
 	// disable this behaviour being added later, once this has stablised.
 	EnableDirtyRuns bool
 
-	// The hostname that will be used to bind the homeserver ports to, e.g. `127.0.0.1`
+	// The IP that will be used to bind the homeserver ports to, e.g. `127.0.0.1`  (only
+	// allow localhost access), `0.0.0.0` (bind to all available interfaces).
+	//
+	// For Complement tests, this is always configured as `127.0.0.1` but can be
+	// overridden by homerunner to allow binding to a different IP address
+	// (`HOMERUNNER_HS_PORTBINDING_IP`).
+	//
+	// This field is also used for the host-accessible homeserver URLs (as the hostname)
+	// so clients in your tests can access the homeserver.
```

Comment:
As long as it's not actually used for binding, I think this is misleading.

```suggestion
	// The IP that is used to connect to the running homeserver from the host.
	//
	// For Complement tests, this is always configured as `127.0.0.1` but can be
	// overridden by homerunner to allow binding to a different IP address
	// (`HOMERUNNER_HS_PORTBINDING_IP`).
	//
	// This field is used for the host-accessible homeserver URLs (as the hostname)
	// so clients in your tests can access the homeserver.
```

Reply:
Updated :white_check_mark: This sounds accurate enough to move on :fast_forward:

We could say "Historically, used for ..."